openapi.json

```diff
@@ -1525,43 +1712,157 @@
             ],
-            "title": "Interrupt Before",
-            "description": "Nodes to interrupt immediately before they get executed."
+            "title": "Stream Mode",
+            "description": "The stream mode(s) to use.",
+            "default": [
+              "values"
+            ]
+          },
+          "stream_subgraphs": {
+            "type": "boolean",
+            "title": "Stream Subgraphs",
+            "description": "Whether to stream output from subgraphs.",
+            "default": false
+          },
+          "on_disconnect": {
+            "type": "string",
+            "enum": [
+              "cancel",
+              "continue"
+            ],
+            "title": "On Disconnect",
+            "description": "The disconnect mode to use. Must be one of 'cancel' or 'continue'.",
+            "default": "cancel"
           },
-          "interrupt_after": {
+          "multitask_strategy": {
+            "type": "string",
+            "enum": [
+              "reject",
+              "rollback",
+              "interrupt",
+              "enqueue"
+            ],
+            "title": "Multitask Strategy",
+            "description": "Multitask strategy to use. Must be one of 'reject', 'interrupt', 'rollback', or 'enqueue'.",
+            "default": "reject"
+          },
+          "if_not_exists": {
+            "type": "string",
+            "enum": [
+              "create",
+              "reject"
+            ],
+            "title": "If Not Exists",
+            "description": "How to handle missing thread. Must be either 'reject' (raise error if missing), or 'create' (create new thread).",
+            "default": "reject"
+          },
+          "after_seconds": {
+            "type": "integer",
+            "title": "After Seconds",
+            "description": "The number of seconds to wait before starting the run. Use to schedule future runs."
+          }
+        },
+        "type": "object",
+        "required": [],
+        "title": "RunCreateStateful",
+        "description": "Payload for creating a run."
+      },
+      "RunCreateStateless": {
+        "properties": {
+          "assistant_id": {
             "anyOf": [
               {
                 "type": "string",
-                "enum": [
-                  "*"
-                ]
+                "format": "uuid",
+                "title": "Assistant Id"
               },
+              {
+                "type": "string",
+                "title": "Graph Id"
+              }
+            ],
+            "description": "The assistant ID or graph name to run. If using graph name, will default to first assistant created from that graph."
+          },
+          "input": {
+            "anyOf": [
               {
                 "items": {
-                  "type": "string"
+                  "type": "object"
                 },
                 "type": "array"
+              },
+              {
+                "type": "object"
```

Comment:
In case of LangGraph Functional API, this can now be anything, not just array of objects or objects

Reply:
That's a good point, will change now